scrapy/utils/iterators.py

```diff
@@ -44,7 +44,7 @@ def xmliter_lxml(obj, nodename, namespace=None, prefix='x'):
     iterable = etree.iterparse(reader, tag=tag, encoding=reader.encoding)
     selxpath = '//' + ('%s:%s' % (prefix, nodename) if namespace else nodename)
     for _, node in iterable:
-        nodetext = etree.tostring(node, encoding='unicode')
+        nodetext = etree.tostring(node, encoding='utf-8')
```

Comment:
`Selector` calls `.encode('utf-8')` on the text attribute in `create_root_node()`, so looks like this is not needed and may be harmful?

Reply:
(see also the comment for docs/topics/dynamic-content.rst)